content/en/platform/corda/5.0/deploying-operating/observability/metrics.md

```diff
@@ -230,3 +235,89 @@ the maximum value, and the cumulative sum of values, respectively.
+| `corda_db_entity_persistence_request_time_seconds` | Timer | <ul><li>`entityRequest_type`</li><li>`entityRequest_outcome`</li></ul> | The time it takes to process an entity persistence request, from the moment the request is received from Kafka. The `entityRequest.type` tag is the type of persistence request, The `entityRequest_outcome` tag is the outcome of processing a request (SUCCESS, FAILURE). |
+| `corda_db_entity_persistence_request_lag_seconds` | Timer | <ul><li>`entityRequest_type`</li></ul> | The lag between the flow putting the entity persistence request to Kafka and the EntityMessageProcessor. |
+| `corda_ledger_persistence_time_seconds` | Timer | <ul><li>`flowId`</li><li>`ledger_type`</li><li>`operation_name`</li></ul> | The time it takes to execute ledger transaction database request against the database. The `flowId` tag represents the flow ID to correlate with the flow. The `ledger_type` can be UTXO or CONSENSUAL. The `operation_name` tag is the persistence operation. |
+| `corda_membership_persistence_transaction_time_seconds` | Timer | <ul><li>`operation_name`</li><li>`group`</li><li>`virtualnode`</li></ul> | The time it takes to execute membership persistence transactions. Excludes time spent acquiring a database connection. |
+| `corda_membership_persistence_handler_time_seconds` | Timer | <ul><li>`operation_name`</li><li>`group`</li><li>`virtualnode`</li></ul> | The time it takes to execute membership persistence handlers. Includes time taken to get database connection and execute the transaction. The `operation_name` tag is the MGM persistence request name/type. The `group` tag is the membership group within which peer-to-peer communication happens. |
+| `corda_membership_registration_handler_time_seconds` | Timer | <ul><li>`operation_name`</li><li>`group`</li><li>`virtualnode`</li></ul> | Registration is broken down into a series of stages, each with its own handler. This metric measures the time taken to execute each stage.  |
+| `corda_db_reconciliation_run_time_seconds` | Timer | <ul><li>`reconciliation_reconciler_type`</li><li>`reconciliation_outcome`</li></ul> | The time needed for a full reconciliation run. The `reconciliation_reconciler_type` tag is the type of reconciler that run, for example, CPI metadata, virtual node metadata. The `reconciliation_outcome` tag is the outcome of a reconciliation run (SUCCESS, FAILURE). |
+| `corda_db_reconciliation_records_count` | Counter | <ul><li>`reconciliation_reconciler_type`</li><li>`reconciliation_outcome`</li></ul></ul> | The number of reconciled records for a reconciliation run. |
+
+#### Membership Worker
+
+The membership worker is responsible for the processing of application network functionality either on behalf of an MGM,
+a network member, or both. For an MGM, examples of this application functionality include:
+* handling incoming registration requests,
+* network management,
+* and ensuring the network participants are all in sync with the latest network data.
```

Comment:
I'm sure you know better than me on this, but just in case, since this is a bullet point list should it be formatted more like this?
```suggestion
* handling incoming registration requests
* network management
* ensuring the network participants are all in sync with the latest network data
```

Reply:
leaving as is